src/Controller/SiteController.php

```diff
@@ -24,4 +31,72 @@ public function index(ServerRequestInterface $request): ResponseInterface
             ]
         );
     }
+
+    public function about(ServerRequestInterface $request): ResponseInterface
+    {
+        return $this->render(
+            'about',
+            [
+                'csrf' => $request->getAttribute(Csrf::REQUEST_NAME)
+            ]
+        );
+    }
+
+    public function contact(
+        ApplicationParameters $app,
+        ContactForm $form,
+        Flash $flash,
+        Mailer $mailer,
+        UrlGeneratorInterface $url,
+        ServerRequestInterface $request
+    ): ResponseInterface {
+        $body = $request->getParsedBody();
+        $method = $request->getMethod();
+
+        if (($method === Method::POST) && $form->load($body) && $form->validate()) {
+            $message = new Message(
```

Comment:
How about moving forming the message, attaching files and sending into `ContactForm::sendMessage()`?

Reply:
done.